### ISSUE TITLE
May the gopherjs-vue  support vue2?

### ISSUE THREAD

**Issue by scbizu** [oskca/gopherjs-vue#4]
Vue2 has updated some of his APIs and Configures ,will this repo support Vue2? 

The change log is [here](https://github.com/vuejs/vue/issues/2873)

**Comment by robvdl** [comment]
I was interested in finding what would be required to upgrade to Vue 2.0 too and thought I could have a go at it.

First of all, replacing vue.inc.js with v2.x doesn't work, far too much has been changed and it seems the data binding is broken.

I tried to follow the upgrade guide step by step as this isn't a JS project so you can't use the automated upgrade tool and every step has to be checked by hand. I followed this guide:

https://vuejs.org/v2/guide/migration.html

However, it seems actually a huge amount has changed, I got about half way down the page and eventually had to give up when I got to the Events section, which it then goes to say the events property is removed, however some things here are using it, so I am not sure how to fix that.

Basically, I need help to do such a refactor, I don't know quite enough about this project to do such a large refactor.

**Comment by awitherow** [comment]
Quite interested in working on this. You guys wanna try to get together and solve this?

@scbizu @robvdl @oskca

**Comment by scbizu** [comment]
@awitherow  I  will have my 15 days vocation next week .But I do not have too much javascript dev experience at all .

**Comment by oskca** [comment]
works now with VueJS 2.0+ (https://github.com/oskca/gopherjs-vue/commit/3684a4d6ce6e3bebcbfedf6e02ff74db4f8ea762), at least the example works :smile: 

But still have some API to translate, I'll try to finish it later

**Comment by oskca** [comment]
Updated to 2.0+, hope you'll like it
